src/main/java/com/mongodb/spark/sql/connector/read/partitioner/PaginateBySizePartitioner.java

```diff
@@ -38,8 +38,7 @@
  * sized chunks:
  *
  * <ul>
- *   <li>{@value PARTITION_FIELD_LIST_CONFIG}: The field to be used for partitioning. Defaults to:
- *       _id.
+ *   <li>{@value PARTITION_FIELD_CONFIG}: The field to be used for partitioning. Defaults to: _id.
```

Comment:
Let's specify that this field must be unique. This is relevant for all existing implementations (whether abstract or not) of `FieldPartitioner`, but is not required by the `FieldPartitioner` itself.

Reply:
Ack